src/views/Admin/Modules/Edit.vue

```diff
@@ -51,15 +51,68 @@
                               variant="link">+ {{ $t('module.edit.newField') }}</b-button>
                   </th>
                 </tr>
-                <tbody>
-                  <tr>
-                    <th colspan="7">{{ $t('module.edit.systemFields') }}</th>
-                  </tr>
-                  <field-row-view v-for="(field, index) in module.systemFields()"
-                                  :field="field"
-                                  :key="index"></field-row-view>
-                </tbody>
               </table>
+              <div class="d-flex mt-5">
+                <table class="w-50">
+                  <tbody>
+                    <tr>
+                      <th colspan="7">{{ $t('module.edit.systemFields') }}</th>
+                    </tr>
+                    <field-row-view v-for="(field, index) in module.systemFields()"
+                                    :field="field"
+                                    :key="index"></field-row-view>
+                  </tbody>
+                </table>
+                <div class="d-flex flex-column w-50">
+                  <b-row align-v="center" class="text-center justify-content-between mt-4">
+                    <b-col>
+                      <div class="mx-auto circle border-primary text-primary mb-5">
+                        <font-awesome-icon v-if="recordListPage" :icon="['fas', 'check']" />
+                        <span v-else>1</span>
+                      </div>
+                      <b-button v-if="!recordListPage" @click="handleRecordListCreation" :disabled="!namespace.canCreatePage" variant="outline-primary">
+                        Create Record List
+                      </b-button>
+                      <confirm v-else
+                               @confirmed="toRecordList('view')"
+                               @canceled="toRecordList('edit')"
+                               :disabled="!namespace.canManageNamespace"
+                               :borderless="false"
+                               variant="primary"
+                               variantOk="primary"
+                               variantCancel="outline-primary"
+                               size="md"
```

Comment:
default size is md, no?

Reply:
Confirm component in common has it set to 'sm'

Reply:
You could add a prop to control this

Reply:
It already has a prop, its called size.